eng/source-build-patches/0033-Build-net4-in-source-build.patch

```diff
@@ -0,0 +1,96 @@
+  /work/.dotnet/sdk/5.0.100-rc.1.20452.10/Microsoft.Common.CrossTargeting.targets(88,5): error MSB4181: The "MSBuild" task returned false but did not log an error. [/work/artifacts/src/runtime.cf258a14b70ad9069470a108f13765e0e5988f51/src/libraries/System.Configuration.ConfigurationManager/ref/System.Configuration.ConfigurationManager.csproj]
+  /work/artifacts/src/runtime.cf258a14b70ad9069470a108f13765e0e5988f51/src/libraries/System.ServiceProcess.ServiceController/ref/System.ServiceProcess.ServiceController.cs(39,43):
+    error CS0433: The type 'EventLog' exists in both 'System.Diagnostics.EventLog, Version=5.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' [/work/artifacts/src/runtime.cf258a14b70ad9069470a108f13765e0e5988f51/src/libraries/System.ServiceProcess.ServiceController/ref/System.ServiceProcess.ServiceController.csproj]
+  /work/.dotnet/sdk/5.0.100-rc.1.20452.10/Microsoft.Common.CrossTargeting.targets(88,5): error MSB4181: The "MSBuild" task returned false but did not log an error. [/work/artifacts/src/runtime.cf258a14b70ad9069470a108f13765e0e5988f51/src/libraries/System.ServiceProcess.ServiceController/ref/System.ServiceProcess.ServiceController.csproj]
+---
+ src/libraries/Directory.Build.props                         | 6 ++++--
+ .../src/Microsoft.Extensions.Configuration.Xml.csproj       | 3 +++
+ .../ref/System.Configuration.ConfigurationManager.csproj    | 2 +-
+ .../ref/System.ServiceProcess.ServiceController.csproj      | 2 +-
+ 4 files changed, 9 insertions(+), 4 deletions(-)
+
+diff --git a/src/libraries/Directory.Build.props b/src/libraries/Directory.Build.props
+index 94606e51460..807578c11ed 100644
+--- a/src/libraries/Directory.Build.props
++++ b/src/libraries/Directory.Build.props
+@@ -27,8 +27,10 @@
+
+   <PropertyGroup>
+     <!-- Build all .NET Framework configurations when net48 is passed in. This is for convenience. -->
+-    <AdditionalBuildTargetFrameworks Condition="'$(BuildTargetFramework)' == 'net48'">net461;net462;net47;net471;net472</AdditionalBuildTargetFrameworks>
+-    <AdditionalBuildTargetFrameworks Condition="'$(DotNetBuildFromSource)' == 'true' and '$(BuildAllProjects)' == 'true'">$(AdditionalBuildTargetFrameworks);netstandard2.0</AdditionalBuildTargetFrameworks>
++    <NETFrameworkTargetFrameworksExcept48>net461;net462;net47;net471;net472</NETFrameworkTargetFrameworksExcept48>
++    <AdditionalBuildTargetFrameworks Condition="'$(BuildTargetFramework)' == 'net48'">$(NETFrameworkTargetFrameworksExcept48)</AdditionalBuildTargetFrameworks>
++    <AdditionalBuildTargetFrameworks Condition="'$(BuildAllProjects)' == 'true'">$(AdditionalBuildTargetFrameworks);netstandard2.0</AdditionalBuildTargetFrameworks>
++    <AdditionalBuildTargetFrameworks Condition="'$(DotNetBuildFromSource)' == 'true'">$(NETFrameworkTargetFrameworksExcept48);net48;netstandard2.0</AdditionalBuildTargetFrameworks>
```

Comment:
This should all be deleted and source build should just build all configurations.